stargz/stargz.go

```diff
@@ -171,6 +171,9 @@ type TOCEntry struct {
 	// Xattrs are the extended attribute for the entry.
 	Xattrs map[string][]byte `json:"xattrs,omitempty"`
 
+	// Checksum stores the OCI checksum for regular files payload.
+	Checksum string `json:"checksum,omitempty"`
```

Comment:
s/Checksum/Digest/, as checksum is for corruption detection, not for content-addressibility?

Reply:
renamed